draft-ietf-masque-connect-ip-dns.md

```diff
@@ -306,15 +267,17 @@ an endpoint to send DNS configuration to its peer.
 DNS_ASSIGN Capsule {
   Type (i) = DNS_ASSIGN,
   Length (i),
-  DNS Configuration (..),
+  DNS Configuration Count (i),
```

Comment:
Why do we need the count? It could allow for future extensibility, but at that point I think it's best to just not have it. If we want additional data we'll send it in a new different capsule